### PR TITLE
draft of 2.0.3 release notes

### PR DESCRIPTION
**Pull request by cmastr**
## Jira Ticket
Before creating this pull request, please make sure you have a separate ticket for the docs team to track their work to review and merge, even for minor edits. If you need assistance, ask in the #doc-team channel on slack.

https://jira.d2iq.com/browse/D2IQ-66070 

## Checklist
- [ ] Make sure you change all affected versions (e.g. 1.10, 1.11, 1.12, 1.13).
- [ ] Test all commands and procedures where applicable.
- [ ] Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects) if you are moving a page.

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing) for more information.

### CODE REVIEW

File: pages/mesosphere/dcos/2.0/release-notes/2.0.3/index.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+## Components
+
+DC/OS 2.0.3 includes the following Marathon&trade; component version:
+
+- Marathon 1.9.136
+
+### Marathon Fixed and Improved Issues
+
+- Improved the expunge logic so that it evaluates in the same timely manner that unreachable inactive evaluates. (COPS-5617)
```
Comment (timcharper) [inline pages/mesosphere/dcos/2.0/release-notes/2.0.3/index.md]: - `/v2/tasks` plaintext output in Marathon 1.5 returned container network endpoints in an unusable way. (MARATHON-8721)
 - Marathon launched too many tasks when a missing docker image is resolved during a deployment. (DCOS_OSS-5679)
 - Marathon would omit pod status report with tasks in `TASK_UNKOWN` state. (MARATHON-8710)
 - Marathon was checking authorization for unrelated apps when performing a kill-and-scale operations; this has been resolved. (MARATHON-8731)

File: pages/mesosphere/dcos/2.0/release-notes/2.0.3/index.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+- Fixed an issue where in some rare circumstances, after upgrading a cluster from DC/OS 1.11 to DC/OS 1.13 users were no longer able to launch tasks that use the UCR containerizer. (D2IQ-64507, COPS-5868)
+
+# Marathon
+
+## Components
+
+DC/OS 2.0.3 includes the following Marathon&trade; component version:
+
+- Marathon 1.9.136
+
```
Comment (timcharper) [inline pages/mesosphere/dcos/2.0/release-notes/2.0.3/index.md]:
```suggestion

- Metronome 0.6.41
```

File: pages/mesosphere/dcos/2.0/release-notes/2.0.3/index.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+# DC/OS 
+
+## Components
+
+DC/OS 2.0.3 includes the following component versions:
+
+- Apache&reg; Mesos&reg; 1.8.2-dev
+- OpenSSL 1.1.1d	
+
+### DC/OS Fixed and Improved Issues
```
Comment (cneth) [inline pages/mesosphere/dcos/2.0/release-notes/2.0.3/index.md]: We're missing an entry for https://jira.d2iq.com/browse/COPS-4995.   The Mesos fix is here: https://github.com/apache/mesos/commit/a2ca451aab4625e126b9e7b470eb9f7c232dd746

File: pages/mesosphere/dcos/2.0/release-notes/2.0.3/index.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
+---
+layout: layout.pug
+navigationTitle: Release notes for 2.0.3
+title: Release notes for 2.0.3
+menuWeight: 0
+excerpt: Release notes for DC/OS 2.0.3, including Open Source attribution, and version policy.
+---
+DC/OS&trade; 2.0.3 was released on 19 March 2020.
```
Comment (cneth) [inline pages/mesosphere/dcos/2.0/release-notes/2.0.3/index.md]: I assume this will be updated? :-)

Comment (cmastr) [reply pages/mesosphere/dcos/2.0/release-notes/2.0.3/index.md]: Yep good catch. I replaced with ?? for placeholder until date is final.

File: pages/mesosphere/dcos/2.0/release-notes/2.0.3/index.md (hunk 3 above)
Comment (cmastr) [reply pages/mesosphere/dcos/2.0/release-notes/2.0.3/index.md]: added

**Pull request merged**